.atomist/tslint.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Looks like the TS guidelines actually suggest to not do this anyway: https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines#names